### PR TITLE
User/alan.lam/display round num

### PR DESCRIPTION
**Pull request by alantylam**

### CODE REVIEW

**Comment by alantylam** [comment]
Arcade - Mozilla Firefox 2020-08-02 12_40_22 AM

File: frontend/src/modules/scribble/components/RoundNumber.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export default {
+    name: "RoundNumber",
+
+    props: {
+        roundNumber: Number,
+    },
+
+    data: function () {
+        return {
+            roundNum: this.roundNumber + 1,
```
Comment (Hansel34) [inline frontend/src/modules/scribble/components/RoundNumber.vue:28]: nit: I would add a comment or use a variable for the 1- the +1 is not obvious to why from the code.

Comment (enochtsang) [reply frontend/src/modules/scribble/components/RoundNumber.vue:28]: As a note, programmers have a term for this, magic numbers! https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

File: frontend/src/modules/scribble/components/RoundNumber.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<template>
+    <div class="round-number">
+        <b-container fluid>
+            <b-row align-v="center" class="justify-content-md-center">
+                <b-col md="auto">Round: </b-col>
+                <b-col md="auto"
+                    ><input
+                        class="round-number__input"
+                        type="text"
+                        v-model="roundNum"
```
Comment (enochtsang) [inline frontend/src/modules/scribble/components/RoundNumber.vue:10]: `v-model` is more for two-way bindings, but in this case it's `readonly` and a little misleading. It'd be better to just use a `<p>` tag and reference the value so a reader knows it's not supposed to be input.

File: frontend/src/modules/scribble/stores/states/gamestates.js [hunk 3]
```diff
@@ -19,11 +19,12 @@ class GameState {
 
 export class ChoosingWord extends GameState {
     static STATE = "ChoosingWord";
```
Comment (enochtsang) [inline frontend/src/modules/scribble/stores/states/gamestates.js:21]: This state is only active for the player choosing the word, so other players don't get this update. You can see it comes up as NaN for other players.
image

File: frontend/src/services/eventHandlers/gameHandler.js [hunk 4]
```diff
@@ -76,6 +77,13 @@ export default class GameHandler {
                 );
                 EventBus.$emit(Event.CORRECT_GUESS, player);
             }
+        } else if (payload.gameMasterAPI === "scoreTime") {
+            store.commit("scribble/setRoundNumber", payload.scoreTime.round);
+        } else if (payload.gameMasterAPI === "requestCurrentGameInfo") {
+            store.commit(
+                "scribble/setRoundNumber",
+                payload.requestCurrentGameInfo.round
```
Comment (enochtsang) [inline frontend/src/services/eventHandlers/gameHandler.js:85]: If this is gonna get implemented, we should update as much as possible here. i.e. try to use all our data available.

**Pull request merged**